rdrf/rdrf/templates/rdrf_cdes/form_error_not_translated.html

```diff
@@ -0,0 +1,11 @@
+{% extends "rdrf_cdes/base.html" %}
+{% load i18n %}
+{% load get_language %}
+
+{% block content %}
+    <div class="alert alert-danger">
+      {% blocktrans trimmed with current_language=LANGUAGE_CODE|get_language %}
+        This form is not available in {{ current_language }}.
```

Comment:
Technically, `Language`'s `__str__` isn't translated to work bidirectionally. Could use the built-in [get_language_info](https://docs.djangoproject.com/en/5.1/topics/i18n/translation/#get-language-info) templatetag here instead, but this might have different names for the language than are defined in settings, so probably best to use gettext on the `__str__` method.

Reply:
Fixed.